Pages/Index.cshtml

```diff
@@ -56,7 +56,7 @@
 
 @if (Model.SearchResults != null)
 {
-    <div class="search-results">
+    <div class="search-results border-dark">
```

Comment:
Does this reduce technical debt?  If so, how so?  Is there an accessibility issue here that this fixes?
Code reviews should focus on alignment with best practices.  Unless there is a tech debt related reason to do so, UI changes should be left up to design.